src/Spryker/Zed/EventBehavior/Communication/Console/EventTriggerConsole.php

```diff
@@ -95,9 +95,9 @@ protected function getResourcesUsageText(): string
         $availableResourceNames = $this->getFacade()->getAvailableResourceNames();
 
         return sprintf(
-            "-%s [\n\t%s\n]",
```

Comment:
This will change console output, is it intentional change?

Reply:
Since your string does not longer contains `\n\t`, please use single quotes here and below

Reply:
Well, this doesn't relate to the fix but it'd rather fix another bug.